images/app/Dockerfile

```diff
@@ -28,4 +28,4 @@ USER app
 
 ENV PATH=/.venv/bin:${PATH}
 
-CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "8000"]
+CMD ["gunicorn", "--bind", "0.0.0.0:8000", "--worker-class", "uvicorn.workers.UvicornWorker", "--workers", "4", "app.main:app"]
```

Comment:
:thinking: It would be cool if workers was also configurable from an environment variable. I wonder if that is possible

Reply:
I'm currently adjusting on the deployment, if necessary, but could find a way here of course :ok_hand:

Reply:
okay @bkochendorfer, starting gunicorn with `config.py` now, building settings with pydantic :slightly_smiling_face: